src/Components/Home.js

```diff
@@ -47,7 +48,12 @@ class Home extends Component{
         playerList={this.state.players}
         gameDS={this.state.data_structure}
         />
-    </Button>
+       </Button>
+
+      <div className='Rankings'>
+          <RankingTable />
+      </div>
+
```

Comment:
I just added the table below the home page options for now. We can discuss on how to better place it on the page